src/views/registration-js/attendees/fields/birth.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/birth.php
+ *
+ * @since   TBD
+ *
+ * @see     Tribe__Tickets_Plus__Meta__Field__Birth
+ */
+
+$required    = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id   = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$birth_field = $field;
+$field       = (array) $field;
+$field_name  = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled    = false;
+$classes     = [
+		'tribe-common-b1',
+		'tribe-field',
+		'tribe-tickets__item__attendee__field__birth',
+		'tribe-tickets-meta-required' => $required,
```

Comment:
:no_entry_sign: Array item not aligned correctly; expected 4 spaces but found 8

<sub>hash: ccc413f9045fc7bfc1f83a44848a5fe3</sub>